src/seedu/addressbook/data/person/Street.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's street in the address book.
+ * Guarantees: immutable
+ */
+public class Street {
+
+    public final String value;
+	
+	/**
+     * Validates given street.
```

Comment:
this is incorrect comment, no?
you are not doing any validation here. you only setting the value.